README.md

```diff
@@ -1,33 +1,7 @@
-# Project
+#### Azure IoT Utility
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+This is the location of the Arduino-specific source files for the
+[AzureIoTUtility Wi-Fi adapter Arduino published library](https://github.com/Azure/azure-iot-arduino-socket-wifi). 
```

Comment:
link broken?

Reply:
ahh yup updated it in the pal but not here